Docs/Configuration.tex

```diff
@@ -265,35 +265,35 @@ \subsection{Configuration Terms}\label{configuration-terms}
   \texttt{invalid\ value} if missing.
 \item
   \texttt{fatal\ behaviour} --- behaviour leading to boot termination.
-  Implementation must stop the boot process from going any further until
-  next host system boot. It is allowed but not required to perform cold
-  reboot or show any warning message.
+  Implementations must prevent the boot process from proceeding until the
+  next host system boot. It is permitted, but not required, to execute
+  cold reboots or to show warning messages in such cases.
 \item
   \texttt{undefined\ behaviour} --- behaviour not prescribed by this
-  document. Implementation is allowed to take any measures including but
-  not limited to \texttt{fatal\ behaviour}, assuming any states or
-  values, or ignoring, unless these measures negatively affect system
-  security in general.
```

Comment:
Suggest these particular three lines are not really improved by the changes, perhaps the only change really needed is "ignoring" -> "ignoring them"?

Reply:
Concur

Reply:
Disagree on review. Will leave for Vit to make the call.

Reply:
Okay, let's try to leave to Vit things where we really can't agree, and where at least one of us thinks something important really hangs on it? Happy to go with your wording here. (And may have been over-zealous in rejecting it, anyway!)

Reply:
I suspect (and hope, for his own sake!) that Vit will be happy to review in much less detail if two of us have had eyes on it and are both happy with most of it!

Reply:
He will review every item but the more eyes the better and he can pay extra attention to some items.

Reply:
cc @vit9696

Reply:
Can live with it…